demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -217,6 +218,23 @@ export function ProductsGrid() {
             valueOptions: ["Cap", "Shirt", "Tie"],
             disableExport: true,
         },
+        {
+            field: "titleSlugOrDescription",
+            headerName: "Title, Slug or Description",
+            width: 150,
+            visible: theme.breakpoints.down(0), // always hidden but used for filtering
+            disableExport: true,
+            filterOperators: getGridStringOperators().filter((operator) => operator.value === "contains"),
+            toGqlFilter: (filterItem) => {
+                return {
+                    or: [
```

Comment:
If i see it correctly, filtering multiple `or`'s on different columns, will override each other?